packages/stark-ui/assets/themes/_app-logo-theme.scss

```diff
@@ -0,0 +1,14 @@
+// Import all the tools needed to customize the theme and extract parts of it
+@import "~@angular/material/theming";
+
+// Define a mixin that accepts a theme and outputs the color styles for the component.
+@mixin nbb-logo-theme($theme, $button-vars) {
+  // Extract whichever individual palettes you need from the theme.
+  $primary: map-get($theme, primary);
+  $accent: map-get($theme, accent);
+
+  // Use mat-color to extract individual colors from a palette as necessary.
+  .stark-app-logo .mat-card {
```

Comment:
I think this file should not be there. It was apparently linked to your custom implementation of app-logo component...

Reply:
Indeed testing code, I keep the structure anyway so if somebody works on it, the sccs files are already there

Reply:
good point, in fact they should be in the same folder of the component.

Then in the starter/showcase, the dev should import those himself

Reply:
And they should be included in the angular.json file ;)

Reply:
Why including css files in the angular.json?